<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test1012Class</class>
 <widget class="QMainWindow" name="test1012Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test1012</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ê»’!!!</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget1">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>71</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test1012.qrc"/>
 </resources>
 <connections/>
</ui>
